web/ui_utils.js

```diff
@@ -306,9 +306,12 @@ function approximateFraction(x) {
   return result;
 }
 
-function roundToDivide(x, div) {
-  const r = x % div;
-  return r === 0 ? x : Math.round(x - r + div);
```

Comment:
This logic was introduced by https://github.com/mozilla/pdf.js/pull/6552. By doing `x - (x % div) + div` we were:
- always rounding up, since we were adding 1div to floor<sub>multiples of div</sub>(x)
- given that `div` is an integer, `x - (x % div)` is a multiple of `div` (thus, an integer) and `x - (x % div) + div` is also an integer, so there is no need to `round()`.

Reply:
And what's the problem with rounding up instead of flooring ?

Reply:
That by rounding up when you are already close to the maximum you end up going above the maximum.